### ISSUE TITLE
compose validators with composed Validators

### ISSUE THREAD

**Issue by lauterry** [jfairbank/revalidate#23]
Hi

I'm wondering if it is possible to compose validators with composed validators ?

```
export const firstnameValidator = composeValidators(
	hasLengthGreaterThan(1)({
		message: ""
	}),
	hasLengthLessThan(31)({
		message: ""
	})
)('firstname');

export const requiredFirstnameValidator = composeValidators(
	isRequired({
		message: ""
	}),
	firstnameValidator
)('firstname');

```

**Comment by jfairbank** [comment]
Hi, @lauterry.

Composed validators should be composable too, but I believe you found a bug. I would expect your example to not work with the empty string for all the messages. However, when I changed the messages to not be empty, it did not correctly validate some test values I tried on it. I'm pretty sure I know what the bug is, so I should have a fix today or tomorrow. Thanks for finding this!

**Comment by jfairbank** [comment]
@lauterry I released a fix in v0.4.1. Try it out and let me know if you have any issues. Thanks again!

**Comment by lauterry** [comment]
@jfairbank Yeahh. Thx. I'll test it as soon as possible and let you know
